### ISSUE TITLE
Paravirt setting causes my vagrant up to error.

### ISSUE THREAD

**Issue by jazzdrive3** [Lullabot/trusty32-lamp#22]
#<Vagrant::Errors::VBoxManageError: There was an error while executing `VBoxManage`, a CLI used by Vagrant
for controlling VirtualBox. The command and stderr is shown below.

Command: ["modifyvm", "b5280b81-e5e6-4d38-b4d4-c30247ef4912", "--paravirtprovider", "default"]

I'm running VirtualBox 4, so should we add to the README that version 5 is a requirement? Or is there a way we can check the virtualbox version and conditionally run commands?

Just commenting out the line for now until I update.

**Comment by deviantintegral** [comment]
Yeah... I suppose I could be a little more explicit in that I only test against the latest releases when building a box: https://github.com/Lullabot/trusty32-lamp#setup-instructions

We probably can do conditional version checks - but is there any reason you can't upgrade to Virtualbox 5? It's had some great performance improvements and didn't break anything for me.

On the bright side, with proper box versioning it should be much simpler to get older versions if needed in the future.

**Comment by deviantintegral** [comment]
This should make it only trigger for 5+:

```
+    # If Virtualbox is version 5.0.0 or higher, enable the kvm paravirtualization
+    # provider for better performance.
+    begin
+      if VagrantPlugins::ProviderVirtualBox::Driver::Meta.new.version >= "5.0.0"
+        v.customize ['modifyvm', :id, '--paravirtprovider', 'kvm']
+      end
+      rescue Vagrant::Errors::VirtualBoxNotDetected
+    end
```

**Comment by makangus** [comment]
I am having the same issue. Confirmed https://github.com/Lullabot/trusty32-lamp/issues/22#issuecomment-142918646 fixes it.

**Comment by makangus** [comment]
Fixed by #25 , closing.
